src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/entities/User.java

```diff
@@ -0,0 +1,19 @@
+package br.com.mercadolivre.bootcampw2.grupo11.socialmeli.entities;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
+@NoArgsConstructor
+@Getter
+@Setter
+public class User {
+    @Id
+    @GeneratedValue
+    private long userId;
```

Comment:
When w e create the user to insert into the database, we don't have the userId, so we let the value to be null
With primitives you cant do that, its initialized with 0L, so its a better choice to use the wrapper
Also, just naming id is more precise, because we already inside user, so there is no need to specify that again.